platform/openide.util/src/org/openide/util/TopologicalSortException.java

```diff
@@ -47,7 +47,7 @@
     /** vertexes sorted by increasing value of y */
     private Stack<Vertex> dualGraph = new Stack<Vertex>();
 
-    TopologicalSortException(Collection vertexes, Map<?,? extends Collection<?>> edges) {
+    TopologicalSortException(Collection<?> vertexes, Map<?,? extends Collection<?>> edges) {
```

Comment:
Is the switch from raw type to generic intended? If so, is the bound planned as is?